test/0.8.9/oracle/hash-consensus-happy-path.test.js

```diff
@@ -21,36 +37,36 @@ contract('HashConsensus', ([admin, member1, member2, member3, stranger]) => {
     const INITIAL_EPOCH = 3
 
     it('deploying hash consensus', async () => {
-      const deployed = await deployHashConsensus(admin, {initialEpoch: INITIAL_EPOCH})
+      const deployed = await deployHashConsensus(admin, { initialEpoch: INITIAL_EPOCH })
       consensus = deployed.consensus
       reportProcessor = deployed.reportProcessor
     })
 
     it('adding members', async () => {
       await consensus.addMember(member1, 1, { from: admin })
       assert.equal(await consensus.getIsMember(member1), true)
-      assert.equal(+await consensus.getQuorum(), 1)
+      assert.equal(+(await consensus.getQuorum()), 1)
```

Comment:
can we allow `+await`? the extra parentheses create a lot of visual noise

Reply:
fixed